keras_nlp/utils/audio_utils.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import numpy as np
+import tensorflow as tf
+
+try:
+    import ffmpeg
+except ImportError:
+    ffmpeg = None
+
+
+def load_audio(file, sample_rate=16000):
```

Comment:
Let's not add this for now (as mentioned above, we should figure out whether this can live in core keras). This can go in our guides and examples for the time being.